py3status/modules/keyboard_layout.py

```diff
@@ -132,6 +133,11 @@ def _setxkbmap(self):
         layouts = re.match(LAYOUTS_RE, out).group(1).split(",")
         if len(layouts) == 1:
             return layouts[0]
```

Comment:
if you return here, how is that the rest of your commit works?

Reply:
I hadn't touched this module so I don't know what the output should look like. I'll ask @inigoserna for `setxkbmap -query` output... and one more output again... with >1 layout if possible?

Reply:
Well I mean, your identation is the same so the return will always happen and your variant handling never be executed no?

Reply:
Sorry, can't understand.
The code I proposed is:
```
         if len(layouts) == 1:
-            return layouts[0]
+            variant = re.match(VARIANTS_RE, out)
+            if variant:
+                return layouts[0] + ' ' + variant.group(1)
+            else:
+                return layouts[0]
```

Reply:
@inigoserna I forget to remove `return layouts[0]` and @ultrabug caught that. Is fixed.